### DISCUSSION TITLE
Text is not positioned correctly inside energy storages GUI

### DISCUSSION THREAD

**Discussion by reoseah** [IndustrialCraftMC/IndustrialCraft2#65] [【漏洞反馈】Bugs Report]
Text is placed on top of the energy meter. See image:
![2023-05-10_15 55 41](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/20323084/7e440f46-d6d9-46e1-b6d5-36ba0f4a970a)

**Comment by AbbottMc** [comment]
yeah it's really annoying
